index.js

```diff
@@ -34,33 +37,36 @@ module.exports = function (options, cb) {
 		0x00, 0x02, /* Type:  NS      */
 		0x00, 0x01  /* Class: IN      */
 	]);
+}
```

Comment:
**Divide and conquer**. 

It's not easy to understand, and you're increasing 10 lines inside a function when a "reader" doesn't need to see  that information (**visual noise**). It's a point of view, if you disagree I can change it.